contrib/clojure-package/README.md

```diff
@@ -14,43 +14,43 @@ For a **video introduction**, see [Clojure MXNet with Carin Meier - Clojure Virt
 
-Testing instructions can be found in the testing.md.
 
 ## Getting Started
 
+The Clojure MXNet framework consists of a core C library, a Scala Api that talks to it through JNI bindings, and finally a Clojure wrapper around the Scala Api.
+
+Since there is a native code involved in the framework, what OS you are running matters.
+
 The following systems are supported:
 
 - OSX cpu
 - Linux cpu
 - Linux gpu
 
-There are two ways of getting going. The first way is the easiest and that is to use the pre-built jars from Maven. The second way is to build from source. In both cases, you will need to load the prereqs and dependencies, (like opencv).
-
+There are three ways of getting started:
 
+* [Use the Clojure jars with the native dependencies baked in](#getting-started-with-the-clojure-jars-on-maven). This the easiest way to get going.
+* [Checkout the MXNet project from a release tag using the Scala jars with native dependencies](#getting-started-with-mxnet-project-with-the-prebuilt-scala-jars). This is also a pretty easy way to get started.
+* [Build from the MXNet project master](#getting-started-with-mxnet-project-with-building-from-source). This option can be used to build the whole project yourself.
 
-### Prerequisites
 
+### Getting Started with the Clojure Jars on Maven
 
-Follow the instructions from https://mxnet.incubator.apache.org/install/osx_setup.html or https://mxnet.incubator.apache.org/install/ubuntu_setup.html
-about _Prepare Environment for GPU Installation_
-and _Install MXNet dependencies_
+This option is one of the fastest. If you are looking to use MXNet in your own Clojure project, this is your best bet.
 
+The Clojure MXNet jars are found on [maven.org](https://search.maven.org/search?q=clojure%20mxnet).
 
-#### Cloning the repo and running from source
+- Create your own project with `lein new my-mxnet`
+- Edit your `project.clj` based on your system with the desired jar from maven. Replace `x.x.x` with the latest version on [maven](https://search.maven.org/search?q=g:org.apache.mxnet.contrib.clojure).
 
-To use the prebuilt jars (easiest), you will need to replace the native version of the line in the project dependencies with your configuration.
+   - [org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-gpu "x.x.x"]
+   - [org.apache.mxnet.contrib.clojure/clojure-mxnet-linux-cpu "x.x.x"]
+   - [org.apache.mxnet.contrib.clojure/clojure-mxnet-osx-cpu "x.x.0"]
 
-`[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-gpu "1.3.0"]`
-or
-`[org.apache.mxnet/mxnet-full_2.11-linux-x86_64-cpu "1.3.0"]`
-or
-`[org.apache.mxnet/mxnet-full_2.11-osx-x86_64-cpu "1.3.0"]`
 
-If you are using the prebuilt jars they may have a slightly different dependencies then building from source:
+You also might need the following dependencies installed depending on your system because of the native dependencies assumed in the clojure-mxnet jars.
 
 *For OSX you will need:*
 
```

Comment:
I think we also need `brew install wget` as wget isn't installed by default on osx. 

```suggestion
`brew install wget`
```

Reply:
Nice! Thanks @susanluke